### ISSUE TITLE
Dependency upgrade

### ISSUE THREAD

**Issue by ManuelRauber** [icapps/flutter_secure_file_storage#27]
Hi!

Thanks for your package!

Can you take a look at the dependencies and upgrade them to the latest version?

Especially

```yaml
flutter_key_value_file_storage: ^0.0.6
flutter_secure_storage: ^8.0.0
```

do have newer versions, see [flutter_key_value_file_storage](https://pub.dev/packages/flutter_key_value_file_storage) and [flutter_secure_storage](https://pub.dev/packages/flutter_secure_storage).

Thanks! :)

**Comment by ikbendewilliam** [comment]
@ManuelRauber Hi! I'll put it on the list, feel free to create a PR if this is time sensitive!

**Comment by ikbendewilliam** [comment]
@ManuelRauber this has been released in flutter_secure_file_storage 1.1.0
